crypto/keys/keys_test.go

```diff
@@ -324,3 +324,25 @@ func TestGenerateKey(t *testing.T) {
 		}
 	}
 }
+
+func TestMarshalPrivateKey(t *testing.T) {
+	for _, tc := range []struct {
+		pem     string
+		typeURL string
+	}{
+		{ecdsaPrivateKey, "type.googleapis.com/keyspb.PrivateKey"},
```

Comment:
nit: Please use field keys, it makes future changes less painful.

`{pem: ecdsaPrivateKey, typeURL: "..."},`